tests/cypress/support/commands.js

```diff
@@ -124,17 +124,26 @@ Cypress.Commands.add('getJobNum', (jobID) => {
         });
 });
 
-Cypress.Commands.add('openJob', (jobID = 0) => {
+Cypress.Commands.add('openJob', (jobID = 0, removeAnnotations = true) => {
```

Comment:
Just suggestion to make working with arguments possibly easier:

```js
 Cypress.Commands.add('openJob', ({jobID, removeAnnotations} = {}) => { 
    const $jobID = jobID || 0;  // default value
    const $removeAnnotations = removeAnnotations || true; // default value

    ...
});

```

You can call the command in any ways:
```js
cy.openJob();
cy.openJob({ removeAnnotations: false });
cy.openJob({ jobID: 0 });
``` 

and you do not have to remember order of arguments

Sure, it is more applicable when you have a lot of arguments.